src/style.css

```diff
@@ -0,0 +1,98 @@
+body {
+  margin: 0;
+}
```

Comment:
```suggestion

```
do not remove margin for body in this task.
 it is one of reason failed tests.
 compare reference and tests.
 reference has some horizontal space between edge of page and form
![Screenshot from 2023-04-10 21-33-28](https://user-images.githubusercontent.com/33029864/230969106-16eecffa-27ac-45eb-9d5e-ae3cf5084878.png)